tests/functionality/scopflow/selfcheck.cpp

```diff
@@ -191,13 +253,14 @@ struct ScopflowFunctionalityTests
   void run_test_case(Params &params) override {
     PetscErrorCode ierr;
     SCOPFLOW scopflow;
+    char pbuf[PETSC_MAX_PATH_LEN];
```

Comment:
Again I don't think these should live outside of API

Reply:
I got rid of pbuf.